<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.335">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Steven Roberts">
<meta name="dcterms.date" content="2023-04-23">

<title>Enhancing sustainability of shellfish aquaculture through streamlined maturation control</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="WSG-2023-final_files/libs/clipboard/clipboard.min.js"></script>
<script src="WSG-2023-final_files/libs/quarto-html/quarto.js"></script>
<script src="WSG-2023-final_files/libs/quarto-html/popper.min.js"></script>
<script src="WSG-2023-final_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="WSG-2023-final_files/libs/quarto-html/anchor.min.js"></script>
<link href="WSG-2023-final_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="WSG-2023-final_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="WSG-2023-final_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="WSG-2023-final_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="WSG-2023-final_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
</head><body class="fullcontent">\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhead{}
\fancyfoot{}
\fancyhead[L]{\leftmark}
\fancyhead[R]{\thepage}
\usepackage{fontspec}
\setmainfont{Roboto}
\setsansfont{Open Sans}
\setmonofont{Fira Code}
\usepackage{xcolor}
\definecolor{linkcol}{RGB}{0,0,255}
\definecolor{citecol}{RGB}{0,128,0}
\definecolor{urlcol}{RGB}{255,0,255}
\hypersetup{ linkcolor=linkcol, citecolor=citecol, urlcolor=urlcol, colorlinks=true, }






<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Enhancing sustainability of shellfish aquaculture through streamlined maturation control</h1>
<p class="subtitle lead">Final Report</p>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Steven Roberts </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">April 23, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<p>Summary</p>
<p>1. Characterize genomic processes involved in germ cell specification in Pacific oysters</p>
<p>- Collect transcriptomic data from individual cells from a time series of developing Pacific oyster embryos (year 1)</p>
<p>- Analyze expression profiles of bioinformatically ordered cells to determine the temporal succession of genes involved in germ cell development (year 1 &amp; 2)</p>
<p>2. Optimize delivery techniques of custom gene-regulating molecules (CRISPR-Cas9 constructs or morpholinos) to oyster embryos (year 2)</p>
<section id="characterize-genomic-processes-involved-in-germ-cell-specification-in-pacific-oysters" class="level2">
<h2 class="anchored" data-anchor-id="characterize-genomic-processes-involved-in-germ-cell-specification-in-pacific-oysters">Characterize genomic processes involved in germ cell specification in Pacific oysters</h2>
</section>
<section id="rational" class="level1">
<h1>rational</h1>
<p>Primordial germ cells (PGCs) are essential for the development of reproductive cells like eggs and sperm. They form early in an embryoâ€™s development and eventually give rise to germinal stem cells. There are different ways that PGCs are specified in organisms, which can lead to varying reproductive mechanisms. Understanding how PGCs work is important for managing reproduction in the aquaculture industry, especially in producing sterile fish.</p>
<p>In bivalves, such as oysters, the study of PGCs is still in its early stages. Researchers have identified a gene called vasa, which is important for PGC development in these animals. However, more research is needed to understand other genes involved in this process.</p>
<p>Studying PGCs has been challenging due to the difficulty of isolating these cells from other cell types in embryos. Recent advancements in single-cell RNA sequencing technology offer a new approach to studying PGCs by allowing researchers to analyze individual cells. This method is particularly useful for studying PGCs in bivalves, as they make up only a small number of cells in developing embryos. By applying this technology to early Pacific oyster embryos, researchers hope to gain a better understanding of the genes involved in PGC development in bivalves.</p>
<section id="summary" class="level2">
<h2 class="anchored" data-anchor-id="summary">#summary</h2>
<p>The study involves two experiments on Pacific oyster PGCs (Primordial Germ Cells) to identify and characterize genes uniquely expressed in these cells. The first experiment focuses on the earliest developmental stages, from the first cell division through the blastula stage. These stages were chosen as previous research has shown that the germ line marker, vasa, is restricted to a small number of cells during these stages. The second experiment targets the gastrula stage embryos, which represent a more differentiated embryonic stage, but still containing a small enough number of cells per embryo to sequence representative germ cells.</p>
<p>Researchers collected samples of embryos from various developmental stages, including cleavage, blastula, and gastrula stages. They prepared two batches of embryos for cell dissociation and library preparation. The first batch represented a pool of the first eight samples, while the second batch represented the blastula stage. In a separate experiment, gastrula stage embryos were collected and prepared for cell dissociation.</p>
<p>Cells were dissociated using enzymatic digestion and prepared for sequencing using the Chromium platform. The resulting single-cell mRNA libraries were sequenced, and the data was processed using Cell Ranger software. UMAP (Uniform Manifold Approximation and Projection) was used for visualization and clustering of the cells, with marker gene identification performed using the Monocle3 software. Lastly, an iterative BLAST comparison was used for comparative annotation of the C. gigas gene set with model invertebrates, such as C. elegans, D. melanogaster, and S. purpuratus.</p>
<p>Resjult</p>
<p>Scientists conducted a study to better understand the development of reproductive cells in Pacific oysters. They focused on the role of a gene called vasa, which is known to be important for the development of primordial germ cells (PGCs), the cells that eventually give rise to eggs and sperm.</p>
<p>Using advanced sequencing techniques, researchers analyzed thousands of cells from early-stage oyster embryos. They were able to identify different groups of cells based on their gene expression patterns. Among these groups, one cluster of cells showed high levels of vasa expression, suggesting they were likely to be PGCs or their precursors.</p>
<p>In addition to vasa, the study also found other genes that were specifically expressed in these putative PGCs, including a conserved germline marker called nanos. The identification of these markers could help scientists better understand the processes that govern reproductive cell development in oysters and other related species.</p>
<p>Besides focusing on PGCs, researchers also identified other cell types in the developing embryos, such as ciliary cells, muscle cells, and immune cells. This comprehensive analysis of gene expression in early-stage oyster embryos could provide valuable insights into the complex processes that govern the development of these organisms.</p>
<p>The three most important outcomes of these results are:</p>
<ol type="1">
<li><p>Identification of putative primordial germ cells (PGCs): The study identified a cluster of cells with high levels of vasa expression, suggesting they are likely PGCs or their precursors. This finding is crucial for understanding the development of reproductive cells in Pacific oysters and potentially other related species.</p></li>
<li><p>Discovery of specific gene markers for PGCs: In addition to vasa, the researchers found other genes, including a conserved germline marker called nanos, that were specifically expressed in putative PGCs. These markers could help scientists further study the processes governing reproductive cell development and differentiation in oysters.</p></li>
<li><p>Comprehensive analysis of early-stage oyster embryos: The research provided a detailed view of gene expression patterns in early-stage oyster embryos, identifying various cell types such as ciliary cells, muscle cells, and immune cells. This broad understanding of the complex processes involved in oyster development can contribute to our knowledge of marine biology, aquaculture, and conservation efforts.</p></li>
</ol>
<p>The primary goal of the study was to identify genes uniquely expressed in the earliest germ cells in Pacific oysters. The researchers pooled early cleavage stages through blastula stage embryos to analyze gene expression patterns. They found the germ cell marker vasa expressed almost ubiquitously, but the highest expression was observed in older cells, suggesting either sequestration or zygotic expression. Identifying unique genes co-expressed with vasa was challenging. However, the research provided insights into gene expression patterns in early-stage oyster embryos, identifying ciliary cells, muscle cells, immune cells, and other cell types.</p>
<p>The data revealed heterogeneity in cell types from the gastrula sample, with vasa expression restricted to fewer cells. The presumptive primordial germ cells (PGCs) were identified by the co-expression of vasa and nanos. The gene with the highest specificity to the presumptive PGC cluster was unannotated, making it an interesting candidate for further studies.</p>
<p>In conclusion, this dataset demonstrates the utility of single-cell RNA sequencing to study PGC specification in mollusks, an understudied area. The identification of vasa, nanos, and other genes associated with germ cell formation will be useful for future functional studies.</p>
</section>
<section id="optimize-delivery-techniques-of-custom-gene-regulating-molecules-to-oyster-embryos" class="level2">
<h2 class="anchored" data-anchor-id="optimize-delivery-techniques-of-custom-gene-regulating-molecules-to-oyster-embryos">Optimize delivery techniques of custom gene-regulating molecules to oyster embryos</h2>
</section>
</section>
<section id="electro" class="level1">
<h1>Electro</h1>
<p>This scientific protocol aims to study the potential of electroporation to deliver molecules into geoduck embryos. The experiment is divided into three phases:</p>
<p>Phase 0: Determines if geoduck embryos can be plated and kept alive, and if they autofluoresce under various wavelengths.</p>
<p>Phase I: Explores electroporation parameters and the uptake of fluorescein-labeled dextrans. Parameters to explore include dextran concentration, pulse shape, pulse time, number of pulses, buffer conductivity, pulse voltage, and egg density. Indicators to be used are the percentage of fluorescent embryos and viability assay.</p>
<p>Phase II: Aims to introduce silencing molecules into embryos via electroporation, using fluorophore-labeled morpholinos or guide RNAs.</p>
<p>The experimental design is based on treating fertilized eggs with different electroporation protocols and evaluating the outcomes in terms of fluorescence, viability, and development. Control groups are also established to compare results. Imaging challenges have been encountered, and further optimization is needed. The protocol has been repeated with fresh, higher viability gametes to improve results.</p>
<p>The primary accomplishments of this scientific protocol include:</p>
<ol type="1">
<li><p>Successfully plating and keeping geoduck embryos alive for experimentation.</p></li>
<li><p>Evaluating autofluorescence of geoduck embryos under various wavelengths.</p></li>
<li><p>Investigating the electroporation process for introducing molecules into geoduck embryos.</p></li>
<li><p>Identifying and exploring key electroporation parameters that influence the uptake of fluorescein-labeled dextrans in geoduck embryos.</p></li>
<li><p>Evaluating the potential of electroporation for introducing silencing molecules (fluorophore-labeled morpholinos or guide RNAs) into embryos.</p></li>
<li><p>Comparing the outcomes of different electroporation protocols in terms of fluorescence, viability, and development.</p></li>
</ol>
<p>While some challenges were encountered in terms of imaging and optimization, the protocol has made progress in understanding the potential of electroporation for delivering molecules into geoduck embryos.</p>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "î§‹";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>